articles/embedded.md

```diff
@@ -67,14 +69,17 @@ To answer these questions, both the OFERA EU project as well as several others h
 | A-BUILD-META | Q-BUILD | Meta-Build | Explores a meta-build approach to transform ROS 2 CMakeLists.txt to RTOS-specific build instructions. | TODO |
 | A-BUILD-NUTTX | Q-BUILD | NuttX-specific build | OFERA has integrated (parts of) micro-ROS directly as an app in the NuttX build. |TODO |
 | A-BUILD-ARDUINO | Q-BUILD | Arduino Build | Robotis has explored building all the libraries using the Arduino IDE. This required some manual changes and thus does not scale, but can get you off the ground. | TODO |
+| A-BUILD-NUTTX-LIB | Q-BUILD | NuttX-specific build as library | OFERA has integrated (parts of) micro-ROS directly as a library in the NuttX build. |TODO |
```

Comment:
I see that there is a difference between app- and libary-based integration, but maybe we should update A-BUILD-NUTTX to include both, rather than have two items?

Reply:
Agreed. https://github.com/microROS/design/pull/6/commits/08d7cf33821f9dacc2053042688112b0e3db37e1.

@iluetkeb, let me know if that's acceptable. Feel free to commit on top if you have further suggestions.